### ISSUE TITLE
DeprecationWarning for  photoset = rsp.getchildren()[0]

### ISSUE THREAD

**Issue by shinokada** [sybrenstuvel/flickrapi#131]
When I run pytest I get the following error.

```
tests/test_main.py::test_flickr_search
  /Users/shinokada/.pyenv/versions/3.8.6/envs/mytool/lib/python3.8/site-packages/flickrapi/core.py:690: DeprecationWarning: This method will be removed in future versions.  Use 'list(elem)' or iteration over elem instead.
    photoset = rsp.getchildren()[0]
```

I thought it is a good idea to update it.

**Comment by jim-easterbrook** [comment]
See #130.

**Comment by randomcascade** [comment]
Should be resolved as of #130.
